scopes/react/react/webpack/webpack.config.ts

```diff
@@ -183,6 +183,7 @@ export default function (workspaceDir: string, targets: string[], envId: string,
       // this is for resolving react from env and not from consuming project
       alias: {
         react: require.resolve('react'),
+        '@teambit/ui.mdx-scope-context': require.resolve('@teambit/ui.mdx-scope-context'),
```

Comment:
add `import '@teambit/ui.mdx-scope-context'` in the top, to make sure it will become a dep

Reply:
tx!